apps/demos/Demos/HtmlEditor/MarkdownSupport/Vue/index.html

```diff
@@ -10,26 +10,8 @@
     <script src="../../../../node_modules/typescript/lib/typescript.js"></script>
     <script type="module">
       import * as vueCompilerSFC from "../../../../node_modules/@vue/compiler-sfc/dist/compiler-sfc.esm-browser.js";
-
```

Comment:
please discard this diff, we have this copy pasted in all vue demos, it will be easier to remove it using identical occurrence / regex